src/foremast/templates/infrastructure/iam/kms.json.j2

```diff
@@ -0,0 +1,15 @@
+        {
+            "Sid": "KMSDecrypt",
+            "Action": [
+                "kms:Decrypt"
+            ],
+            "Effect": "Allow",
+            "Resource": [
+            {%- for key_id in items %}
```

Comment:
Where is `items` coming from?

Reply:
Relying on the internal services code that creates items:
https://github.com/foremast/foremast/blob/432284ae32b7f30df99d1bb5eae0a393f9154dd1/src/foremast/iam/construct_policy.py#L111

Adding to our internal templates is fine, but just wanted to release a way of doing it to the community. :)

Reply:
A good example of it used today is: https://github.com/foremast/foremast/blob/master/src/foremast/templates/infrastructure/iam/dynamodb.json.j2